src/leaflet.filelayer.js

```diff
@@ -105,6 +105,24 @@
             return reader;
         },
 
+        loadMultiple: function (files, ext) {
+            var reader = false;
+            if (files[0]) {
+              files = Array.prototype.slice.apply(files);
+              var loadOne = L.Util.bind(function () {
+                reader = this.load(files.shift(), ext, reader);
+                if (files.length > 0) {
+                    setTimeout(loadOne, 25);
```

Comment:
This was in the original FileLayerLoad._loadFiles() method, I just moved it.
But you point is good, and I actually tested without the timeout, it works fine. Code is now cleaner (no more extra 'reader' passed to load() for tests).
I just hope this timeout was not introduced for some good reason... All tests are passing though.